Src/Notion.Client/Api/Databases/RequestParams/DatabasesQueryParameters.cs

```diff
@@ -8,5 +8,16 @@ public class DatabasesQueryParameters : IDatabaseQueryBodyParameters
         public List<Sort> Sorts { get; set; }
         public string StartCursor { get; set; }
         public string PageSize { get; set; }
+
+        public DatabasesQueryParameters CreateCopy()
```

Comment:
If this is only required by extensions then you can move it there. Apart from that we may need deep copy of an object. [ref](https://docs.microsoft.com/en-us/dotnet/api/system.object.memberwiseclone?view=net-5.0)

Reply:
Yep, just for extension